python/cudf/cudf/tests/test_reshape.py

```diff
@@ -114,3 +115,70 @@ def test_df_stack(nulls, num_cols, num_rows, dtype):
 
     assert_eq(expect, got)
     pass
+
+
+def _prepare_merge_sorted_test(
+    size, nparts, keys, na_position, ascending, series=False
+):
+    # TODO: Add null values for `na_position` testing
```

Comment:
Note: Adding null values here seems to cause test failures for some `ascending=False` cases - Not sure if the problem is on the python/cython side (need to investigate further).